specification/purview/data-plane/Azure.Analytics.Purview.Share/preview/2022-06-30-preview/share.json

```
@@ -0,0 +1,1796 @@
+      }
+    },
+    "/receivedShares/attached": {
+      "get": {
+        "tags": [
+          "ReceivedShare"
+        ],
+        "summary": "Get a list of attached received shares.",
+        "description": "List attached received shares",
+        "operationId": "ReceivedShares_GetAllAttached",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/referenceName"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/skipToken"
+          },
+          {
+            "$ref": "#/parameters/$filter"
+          },
+          {
+            "$ref": "#/parameters/$orderby"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/ReceivedShareList"
+            }
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/PurviewShareError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ReceivedShares_GetAllAttached": {
+            "$ref": "./examples/ReceivedShares_GetAllAttached.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/receivedShares/detached": {
+      "get": {
+        "tags": [
+          "ReceivedShare"
+        ],
+        "summary": "Get a list of detached received shares.",
+        "description": "List detached received shares",
+        "operationId": "ReceivedShares_GetAllDetached",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/skipToken"
+          },
+          {
+            "$ref": "#/parameters/$filter"
+          },
+          {
+            "$ref": "#/parameters/$orderby"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/ReceivedShareList"
+            }
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/PurviewShareError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ReceivedShares_GetAllDetached": {
+            "$ref": "./examples/ReceivedShares_GetAllDetached.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/sentShares": {
+      "get": {
+        "tags": [
+          "SentShare"
+        ],
+        "summary": "Get a list of sent shares.",
+        "description": "List sent shares",
+        "operationId": "SentShares_GetAll",
```

The verb in the operationId for `get` on a collection endpoint should be "list".
```suggestion
        "operationId": "SentShares_List",
```